pkg/controller/gitopsconfig/controller_test.go

```diff
@@ -453,6 +451,19 @@ func TestCreatingDeleteJob(t *testing.T) {
 		log.Error(err, "Update CRD", "Failed Updating CRD type of GitOpsConfig")
 	}
 
+	// fakeclient is not updating the job status , inorder to create the job with different action
+	//updating the job status manually
```

Comment:
Please also add information here explaining that this job we are updating exists because it was created by  Reconcile.

Reply:
changes done